### DISCUSSION TITLE
Firebase and Cloud Messaging is supported for mac os application?

### DISCUSSION THREAD

**Discussion by Adbhutashra** [invertase/react-native-firebase#7961] [General]
"@react-native-firebase/app": "^20.3.0",
"@react-native-firebase/messaging" : "^20.3.0",

I am using the latest version of firebase but getting errors:
- 'fetchAssetsWithALAssetURLs:options:' is unavailable: not available on macOS
- 'UIKit/UIKit.h' file not found.

I just want to ensure that firebase and cloud messaging is supported for mac os desktop application?

**Comment by regulusdigitaldev** [comment]
I'm receiving the same 'UIKit/UIKit.h' file not found

"@react-native-firebase/app": "^21.6.1",
"@react-native-firebase/messaging": "^21.6.1"

Haven't updated it yet, as I can see the updated code seems to still attempt to import UIKit/UIKit.h
